src/kernel/src/process/mod.rs

```diff
@@ -421,7 +421,7 @@ impl VProc {
     }
 
     fn sys_thr_set_name(self: &Arc<Self>, i: &SysIn) -> Result<SysOut, SysErr> {
-        let tid: i32 = i.args[0].try_into().unwrap();
+        let tid: i64 = i.args[0].try_into().unwrap();
```

Comment:
This is an infallible conversion, it should be .into(). You'll have to write a From implementation for this I think

Reply:
There already is a From instance for that 👍

Reply:
Oh right, I actually wrote it, but I thought it was in one of the yet unmerged PRs :D